modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/background/task/CTPublishBackgroundTaskExecutor.java

```diff
@@ -247,30 +248,33 @@ public BackgroundTaskDisplay getBackgroundTaskDisplay(
 	public String handleException(
 		BackgroundTask backgroundTask, Exception exception) {
 
-		long fromCTCollectionId = MapUtil.getLong(
-			backgroundTask.getTaskContextMap(), "fromCTCollectionId");
+		if (FeatureFlagManagerUtil.isEnabled("LPD-8012")) {
```

Comment:
Do an early return if feature flag is not enabled so we can keep the code flat, eliminating the extra indentation.